microsoft/testsuites/ltp/ltp.py

```diff
@@ -346,7 +352,7 @@ def _install(self) -> bool:
         self.node.execute("autoreconf -f", cwd=ltp_path, sudo=True)
         make.make("autotools", cwd=ltp_path, sudo=True)
         self.node.execute("./configure --prefix=/opt/ltp", cwd=ltp_path, sudo=True)
-        make.make("all", cwd=ltp_path, sudo=True)
+        make.make("all", cwd=ltp_path, sudo=True, timeout=self.MAKE_TIMEOUT)
```

Comment:
When is it needed? what happened if it's timed out?

Reply:
I was seeing timeouts on WSL, so decided to increase the default value

Reply:
So, if it's timed out, what's error message here before? And after added this, what's expected?

Reply:
There is no error message. Just timeout because it is sometimes slow to make on WSL. So I have increased timeout